// some-lib/bundles/some-lib.umd.js
(function (global, factory) {
    typeof exports === 'object' && typeof module !== 'undefined' ? factory(exports, require('@angular/core')) :
    typeof define === 'function' && define.amd ? define('some-lib', ['exports', '@angular/core'], factory) :
    (global = typeof globalThis !== 'undefined' ? globalThis : global || self, factory(global['some-lib'] = {}, global.ng.core));
}(this, (function (exports, i0) { 'use strict';

    var SomeLibService = /** @class */ (function () {
        function SomeLibService() {
        }
        return SomeLibService;
    }());
    SomeLibService.ɵprov = i0.ɵɵdefineInjectable({ factory: function SomeLibService_Factory() { return new SomeLibService(); }, token: SomeLibService, providedIn: "root" });
    SomeLibService.decorators = [
        { type: i0.Injectable, args: [{
                    providedIn: 'root'
                },] }
    ];
    SomeLibService.ctorParameters = function () { return []; };

    var SomeLibComponent = /** @class */ (function () {
        function SomeLibComponent() {
        }
        SomeLibComponent.prototype.ngOnInit = function () {
        };
        return SomeLibComponent;
    }());
    SomeLibComponent.decorators = [
        { type: i0.Component, args: [{
                    selector: 'my-org-some-lib',
                    template: "\n    <p>\n      some-lib works!\n    </p>\n  "
                },] }
    ];
    SomeLibComponent.ctorParameters = function () { return []; };

    var SomeLibModule = /** @class */ (function () {
        function SomeLibModule() {
        }
        return SomeLibModule;
    }());
    SomeLibModule.decorators = [
        { type: i0.NgModule, args: [{
                    declarations: [SomeLibComponent],
                    imports: [],
                    exports: [SomeLibComponent]
                },] }
    ];

    /*
     * Public API Surface of some-lib
     */

    /**
     * Generated bundle index. Do not edit.
     */

    exports.SomeLibComponent = SomeLibComponent;
    exports.SomeLibModule = SomeLibModule;
    exports.SomeLibService = SomeLibService;

    Object.defineProperty(exports, '__esModule', { value: true });

})));
