// some-lib/fesm2015/some-lib.js
import { ɵɵdefineInjectable, Injectable, Component, NgModule } from '@angular/core';

class SomeLibService {
    constructor() { }
}
SomeLibService.ɵprov = ɵɵdefineInjectable({ factory: function SomeLibService_Factory() { return new SomeLibService(); }, token: SomeLibService, providedIn: "root" });
SomeLibService.decorators = [
    { type: Injectable, args: [{
                providedIn: 'root'
            },] }
];
SomeLibService.ctorParameters = () => [];

class SomeLibComponent {
    constructor() { }
    ngOnInit() {
    }
}
SomeLibComponent.decorators = [
    { type: Component, args: [{
                selector: 'my-org-some-lib',
                template: `
    <p>
      some-lib works!
    </p>
  `
            },] }
];
SomeLibComponent.ctorParameters = () => [];

class SomeLibModule {
}
SomeLibModule.decorators = [
    { type: NgModule, args: [{
                declarations: [SomeLibComponent],
                imports: [],
                exports: [SomeLibComponent]
            },] }
];

/*
 * Public API Surface of some-lib
 */

/**
 * Generated bundle index. Do not edit.
 */

export { SomeLibComponent, SomeLibModule, SomeLibService };
